hw3/01-characters.js

```diff
@@ -2,3 +2,55 @@
 const url = 'https://thronesapi.com/api/v2/Characters';
 
 fetch(url);
+
+const characterFlex = document.getElementById("charcter-items");
+fetch(url)
+  .then((response) => response.json())
+  .then((data) => {
+    data.forEach((c) => {
+        let characterCard = document.createElement("div");
+        characterCard.id = "thrones-characters";
+        characterCard.classList = "col p-2 bg-transparent border-0 mx-auto";
+
+        let characterImageDiv = document.createElement("div");
+        characterImageDiv.classList = "row h-75";
+
+        let characterImage = document.createElement('img');
+        characterImage.id = "thrones-img";
+        characterImage.classList = "h-100";
+        characterImage.alt = "image";
```

Comment:
This may pass some accessibility audits, but it is not accessible. The alternative text should describe the image.